src/python/pants/backend/url_handlers/s3/register.py

```diff
@@ -48,7 +53,44 @@ async def access_aws_credentials() -> AWSCredentials:
         )
         raise
 
-    session = session.get_session()
```

Comment:
was previously shadowing the import so I updated the import to an alias to avoid confusion